composer.json

```diff
@@ -78,5 +78,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "@php php-cs-fixer fix --dry-run --diff",
+        "cs:fix": "@php php-cs-fixer fix",
+        "dev-tools:check": [
+            "./dev-tools/check_file_permissions.sh",
+            "./dev-tools/check_trailing_spaces.sh"
+        ],
+        "dev-tools:docs": "@php dev-tools/doc.php",
+        "dev-tools:install": "@composer -d dev-tools install",
+        "docs": "@dev-tools:docs",
+        "mess-detector": "@php dev-tools/vendor/bin/phpmd . ansi phpmd.xml",
+        "phpstan": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse",
```

Comment:
ok, so have
- `dev-tools:foo` and `foo`
- `dev-tools:bar` but no `bar`
- no `dev-tools:baz: but `baz

maybe it make sense to have only one of them, and consider aspect of being installed behind `dev-tools` or not as detail that doesn't need to be exposed to user?

Reply:
Addressed in #7214